client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/index.tsx

```diff
@@ -11,18 +12,27 @@ import Sidebar from './sidebar';
 import type { Step } from '../../types';
 import './style.scss';
 
-const Launchpad: Step = ( { navigation } ) => {
+type StepContentProps = {
+	siteSlug: string | null;
+	submit: NavigationControls[ 'submit' ];
+};
+
+type LaunchpadProps = {
+	navigation: NavigationControls;
+};
+
+const StepContent = ( { siteSlug, submit }: StepContentProps ) => (
```

Comment:
Obvious not a big deal, but I wonder if there's a point where we consider moving StepContent to its own file. I think it's common in stepper to have this in the same file where the step is defined. But all things equal, I generally prefer the one-component-per-file approach for clean organization.

Reply:
> Obvious not a big deal, but I wonder if there's a point where we consider moving StepContent to its own file. I think it's common in stepper to have this in the same file where the step is defined. But all things equal, I generally prefer the one-component-per-file approach for clean organization.

I actually prefer keeping things consolidated in one file for trivial components so folks don't have to open multiple tabs. That being said, I don't feel strongly either way, so I'll make the change. 👍